test/model/binding.dart

```diff
@@ -231,7 +231,7 @@ class TestZulipBinding extends ZulipBinding {
 
   /// The value that `ZulipBinding.instance.packageInfo` should return.
   PackageInfo packageInfoResult = _defaultPackageInfo;
-  static const _defaultPackageInfo = PackageInfo(version: '0.0.1', buildNumber: '1');
+  static final PackageInfo _defaultPackageInfo = eg.packageInfo();
```

Comment:
nit: omit type where it's obvious (as it is here)

(That doesn't apply to mutable fields/variables, because when it's mutable the initializer at the definition site isn't the last word on its value and so it's never obvious from the definition what the type should be. It does apply to things declared final or const.)